docs/git-compatibility.md

```diff
@@ -109,6 +93,67 @@ https://github.com/octocat/Hello-World` will clone GitHub's "Hello-World" repo
 into a directory by the same name.
 
 
+## Co-located Jujutsu/Git repos
+
+A "co-located" Jujutsu repo is a hybrid Jujutsu/Git repo. These can be created
+if you initialize the Jujutsu repo in an existing Git repo by running `jj init
+--git-repo=.` or with `jj git clone --colocate`. The Git repo and the Jujutsu
+repo then share the same working copy. Jujutsu will import and export from and
+to the Git repo on every `jj` command automatically.
+
+This mode is very convenient when tools (e.g. build tools) expect a Git repo to
+be present.
+
+It is allowed to mix `jj` and `git` commands in such a repo in any order.
+However, it may be easier to keep track of what is going on if you mostly use
+read-only `git` commands and use `jj` to make changes to the repo. One reason
+for this (see below for more) is that `jj` commands will often put the git repo
+in a "detached HEAD" state, since in `jj` there is not concept of a "currently
```

Comment:
I replied via email, and the discussion leaked to the main thread below.

Reply:
> So, a word like "often", "usually", "generally" seemed appropriate.

As we've realized `jj` moves you quite liberally off branches at the moment, I'd still suggest the broad term. I don't want users to have the wrong expectation, if we can't enumerate all the commands which move you off a branch.

Reply:
Can we settle on "usually"? I feel that without a qualifier, we'd be implying that this will *always* happen, which is also not the case.

OTOH, if you feel that we shouldn't have a qualifier regardless, I can go with that.

Reply:
"usually" is fine for me. 

> I feel that without a qualifier, we'd be implying that this will _always_ happen, which is also not the case.

yes, that's a good point.